tests/test_build.nf.test

```diff
@@ -11,7 +11,8 @@ nextflow_pipeline {
 
         when {
             params {
-                outdir               = "$outputDir"
+                outdir = "$outputDir"
+                tools  = "salmon,fusioninspector"
```

Comment:
I think the test might be failing due to a mismatch between the tools specified here and those defined in the profile.

I'd suggest not adding tool parameters directly in the nf-test file, since it relies on the `test_build` profile. The tools used should be defined within that profile, otherwise, the params here would override the profile settings.

The `test_build.config` lists all the processes being tested for traceability, so keeping everything aligned there might help with maintainability.

Reply:
Thanks I didn't realize there was a separate config for that test :) I'll update it!